src/cfnlint/rules/resources/lmbd/SnapStartSupported.py

```diff
@@ -0,0 +1,54 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+from cfnlint.rules import CloudFormationLintRule, RuleMatch
+
+
+class SnapStartSupported(CloudFormationLintRule):
```

Comment:
curious people's thoughts on if we want this lint or not.  the set of runtimes supporting snapstart may change in the future.  do we want to have to change this lint as that happens?  should we instead just let the service be the authority on which runtimes support the feature?

Reply:
cfn-lint is used to identify issues before they are implemented. Since a template with the improper configuration will fail at deployment time I would like to have it implemented before I try create/update a stack. 

As the service changes this capability we will update the rule. Plus we provide options to suppress this rule if needed.

Reply:
@greenwoodcm if the service has a spot in which we can pull this information I would gladly do so over configuring this manually.  We pull a lot of data from SSM, pricing APIs, etc to build the data for our rules.